examples/gno.land/r/demo/groups/misc.gno

```diff
@@ -0,0 +1,121 @@
+package groups
+
+import (
+	"std"
+	"strconv"
+	"strings"
+
+	"gno.land/r/demo/users"
+)
+
+//----------------------------------------
+// private utility methods
+// XXX ensure these cannot be called from public.
+
+func getGroup(gid GroupID) *Group {
+	gidkey := groupIDKey(gid)
+	group_, exists := gGroups.Get(gidkey)
+	if !exists {
+		return nil
+	}
+	group := group_.(*Group)
+	return group
+}
+
+func incGetGroupID() GroupID {
+	gGroupsCtr++
+	return GroupID(gGroupsCtr)
+}
+
+func padLeft(str string, length int) string {
+	if len(str) >= length {
+		return str
+	} else {
+		return strings.Repeat(" ", length-len(str)) + str
+	}
```

Comment:
You can ditch `else` here.
```go
if len(str) >= length {
	return str
}

return strings.Repeat(" ", length-len(str)) + str
```

Reply:
Or you can just:
```go
func padLeft(str string, length int) string {
    return fmt.Sprintf("%*s", lenght, str) 
}
```

Reply:
I tried this, but whenever try to deploy package i get following error.
![image](https://user-images.githubusercontent.com/10301503/219747530-d994959f-427d-42c3-9515-413379a7f589.png)

Reply:
Try swapping out `fmt` for `ufmpt` (it's a gno package) - it should fix your problem